### ISSUE TITLE
HTTPS setup documentation out of date

### ISSUE THREAD

**Issue by Razamuhin** [lennartkoopmann/nzyme#686]
TL;DR: Recommend replacing 'keyUsage = keyEncipherment, dataEncipherment' with 'keyUsage = nonRepudiation, digitalSignature, dataEncipherment' in the documentation's recommended configuration for HTTPS.

Documented instructions for setting up TLS certificates on Nzyme appear to be incorrect/out of date.

The instructions at nzyme.org/docs/installation-configuration/https say to set the keyUsage variable under [v3_req] to keyEncipherment, dataEncipherment as shown:
```
[v3_req]
keyUsage = keyEncipherment, dataEncipherment
```
This configuration throws an ERR_SSL_KEY_USAGE_INCOMPATIBLE error (tested with Brave and Edge browsers).
Instead using
```
[v3_req]
keyUsage = nonRepudiation, digitalSignature, dataEncipherment
```
fixes the issue.

Recommend including the latter variable value in documentation.

**Comment by lennartkoopmann** [comment]
Thank you! Scheduled for one of the coming v2.0 development iterations.

**Comment by lennartkoopmann** [comment]
The entire HTTPs configuration has been simplified and mostly automated in v2.0. Thank you again!
